pkg/label/version.go

```diff
@@ -1,9 +1,13 @@
 package label
 
 const (
+	// AppOperator defines the short name for the app-operator
+	AppOperator = "app-operator"
 	// AppOperatorVersion sets the version label for app custom resources managed
 	// by the operator.
 	AppOperatorVersion = "app-operator.giantswarm.io/version"
+	// CertOperator defines the short name for the cert-operator
+	CertOperator = "cert-operator"
```

Comment:
I think the constants are in the wrong place. Here we usually have the k8s related labels, which is why they are prefixed and have that particular format. I guess it is ok to define the constants in the package where they are used.